src/index.html

```diff
@@ -5,8 +5,314 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#home"><img src="images/logo.png" alt="logo"></a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            class="nav__link is-active"
+            href="#Apple"
+            >Apple</a>
+          </li>
+          <li class="nav__item"><a class="nav__link" href="#Samsung">Samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#Smartphones">Smartphones</a></li>
+          <li class="nav__item">
+            <a
+            data-qa="nav-hover"
+            class="nav__link"
+            href="#Laptops & Computers"
+            >Laptops & Computers</a>
+          </li>
+          <li class="nav__item"><a class="nav__link" href="#Gadgets">Gadgets</a></li>
+          <li class="nav__item"><a class="nav__link" href="#Tablets">Tablets</a></li>
+          <li class="nav__item"><a class="nav__link" href="#Photo">Photo</a></li>
+          <li class="nav__item"><a class="nav__link" href="#Video">Video</a></li>
```

Comment:
It Would be more readable
```suggestion
          <li class="nav__item">
            <a class="nav__link" href="#Samsung">
              Samsung
            </a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#Smartphones">
              Smartphones
            </a>
          </li>
```